### PR TITLE
Add Shiba version 1.2.1

### PR DESCRIPTION
**Pull request by blyoa**
I added a markdown live preview app.

I did not use `extract_dir` because the working directory of the binary file Shiba.exe must be an actual directory instead of a symbolic link.

### CODE REVIEW

File: bucket/shiba.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        }
+    },
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/rhysd/Shiba/releases/download/v$version/Shiba-win32-x64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/rhysd/Shiba/releases/download/$$version/Shiba-win32-ia32.zip"
```
Comment (Ash258) [inline bucket/shiba.json]:
```suggestion
                "url": "https://github.com/rhysd/Shiba/releases/download/v$version/Shiba-win32-ia32.zip"
```

Comment (blyoa) [reply bucket/shiba.json]: Thank you for the review. I updated the PR according to your review.

**Comment by Ash258** [comment]
> I did not use extract_dir because the working directory of the binary file Shiba.exe must be an actual directory instead of a symbolic link.

@r15ch13 Have you tried installation and executing from current / PATH

**Comment by r15ch13** [comment]
@Ash258 yes, I did test it and removed the `extract_dir` again. My fault.

Opened an issue for the symlink problem: https://github.com/rhysd/Shiba/issues/52

**Pull request merged**